### PR TITLE
#166185744 Rework landing page - Give value on the landing page

### PR DESCRIPTION
**Pull request by reifred**
#### What does this PR do?
- Redesign the landing page
#### Description of Task to be completed?
User needs to be able to understand all features of the product on the landing page before scrolling.
### How should this be manually tested?
* After cloning this repo, cd into it and switch to this branch
* Set up the virtual environment and activate it.
* Install the required packages using:
```
pip install -r requirements.txt
```
* Start the development server using ```python manage.py runserver```
* Navigate to the URL ```http://127.0.0.1:8000/```
#### What are the relevant pivotal tracker stories?
[#166185744](https://www.pivotaltracker.com/story/show/166185744)

#### Screenshots (if appropriate)
image
image

### CODE REVIEW

**Review by Emmanuel-Dominic — CHANGES_REQUESTED** [review]
Yes, nice work @reifred but I have cross-checked your PR but i think you need to remove line link `54`  of the `font-awesome` from `template_features.html` because it exists at line `51`.
Thank you.

**Comment by reifred** [comment]
> Yes, nice work @reifred but I have cross-checked your PR but i think you need to remove line link `54` of the `font-awesome` from `template_features.html` because it exists at line `51`.
> Thank you.

Thanks @Emmanuel-Dominic, Let me work on it.

File: requirements.txt [hunk 1]
```diff
@@ -20,8 +20,8 @@ invoke==1.2.0
 requests==2.21.0
 whitenoise==3.3.1
 gunicorn==19.9.0
-psycopg2==2.8.2
-psycopg2-binary==2.8.2
+psycopg2==2.7.7
```
Comment (Rhytah) [inline requirements.txt:23]: @reifred thanks for pinning this, it will make future installs of the package smoother

**Review by Rhytah — APPROVED** [review]

**Comment by Emmanuel-Dominic** [comment]
> > Yes, nice work @reifred but I have cross-checked your PR but i think you need to remove line link `54` of the `font-awesome` from `template_features.html` because it exists at line `51`.
> > Thank you.
> 
> Thanks @Emmanuel-Dominic, Let me work on it.

**Review by Emmanuel-Dominic — APPROVED** [review]
@reifred, great work. I approve you for the requested changes made.

**Comment by Manorlds-Eaglespark** [comment]
Hi @reifred, great work you have done with the landing page. I would recommend you add back the welcome message for the site first-time visitors, that read **WELCOME TO WGER**. This would help the site flow as well.

**Review by Manorlds-Eaglespark — APPROVED** [review]

**Pull request merged**